jme3-plugins/build.gradle

```diff
@@ -11,6 +11,8 @@ sourceSets {
 
 dependencies {
     api project(':jme3-core')
-    api 'com.google.code.gson:gson:2.9.1'
```

Comment:
@pspeed42 I disabled most of Riccardo's changes to work around the bug.

Reply:
> or rather, I don't understand how Ric's changes were working at all before without this.... and if it was working

These changes were removed as a temporary fix; Currently modules `jme3-plugins-json` and `jme3-plugins-json-gson` are not used.

> I don't know why suddenly it needs to change just to fix how .equals()/.hashCode() is working.

Ric's changes are no longer present in the loader (Gltf), it would make no sense to fix something that is not being used, that is why the library imports will restore the connections and make sense of the solution.